docs/sampling/README.md

```diff
@@ -0,0 +1,242 @@
+100% sampling.
+
+When Threshold is not provided, no information about probability
+sampling is available and
+
+### Sampling randomness
+
+When determining the Randomness value from an item of telemetry,
+implementations SHOULD:
+
+- use the `tracestate` OpenTelemetry R-value field (`rv`) if it is present (spans only), or
+- use the `sampling.randomness` attribute value if it is present (logs only), or
+- use the least significant 56 bits of the W3C Trace Context TraceID, as described in the W3C Trace Context Level 2 specification.
+
+In the first two cases, where Randomness is explicitly encoded, the
+value is represented by exactly 14 hexadecimal digits.
+
+Implementations SHOULD NOT require trace flags to have the Trace
+Context Level 2 Random flag set, in case the Trace ID is used as the
+source of randomness.  Because the Random flag is not widely available
+at this time, and because the W3C Trace Context Level 2 specification
+was designed for widespread compliance with existing systems, it is
+recommended to assume there are 56 bits of randomness.
+
+In case a system knowingly uses TraceIDs that do not conform to the
+W3C Trace Context Level 2 specification and they wish to perform
+sampling with OpenTelemetry components, they SHOULD synthesize a
+random R-value and store it in the `tracestate` (Spans) or the
+`sampling.randomness` (Log Records) attribute value.
+
+### No definition for Scope and Resource attributes
+
+We recognize that in some configurations, sampling probability and
+even sampling randomness may be set to a constant value.
+
+The `sampling.threshold` and `sampling.randomness` attributes are not
+defined for use as Scope or Resource attributes in the present
+specification, because it would lead to ambiguity when
+`sampling.priority` is also used.
+
+## Sampling attributes
+
+The following attributes are recognized for Spans and Log Records.
+Since probability sampling is tightly coordinated through Context
+propagation and reflected in the W3C Trace Context `tracestate`, which
+is included in the Span data model, it is not common to express
+sampling randomness and threshold using attributes.
+
+<!-- semconv registry.sampling(omit_requirement_level) -->
+| Attribute  | Type | Description  | Examples  |
+|---|---|---|---|
+| `sampling.priority` | int | Enables prioritizing and de-prioritizing collection of telemetry. [1] | `10`; `1`; `0` |
+| `sampling.randomness` | string | The source of randomness for making probability sampling decisions, when it is not otherwise recorded. [2] | `ce929d0e0e4736` |
+| `sampling.threshold` | string | Sampling probability as specified by OpenTelemetry. [3] | `c`; `ff8` |
+
+**[1]:** This attribute is useful to express a user's desire to collect an item of telemetry, in spite of the default sampling mechanism.  Samplers SHOULD export items with a non-zero value, independent of probability sampling. Samplers SHOULD NOT export items with this attribute set to zero.
```

Comment:
Samplers or exporters?
Also, if we expect samplers to do something about this attribute, it'd be great to clarify that it applies to tail-based sampling

Reply:
I've added `sampling_relevant: true`. There are normative statements about how this should be used in samplers, and I expect them to be used in both head- and tail-based scenarios. The collector's `probabilisticsamplerprocessor` uses this field as specified to bypass its intermediate sampling decisions.

Reply:
I added an example tail-sampler scenario explaining how these attributes might be synthesized, next to the current head-sampler examples.